WordPressAuthenticator/GoogleSignIn/IDToken.swift

```diff
@@ -0,0 +1,16 @@
+/// See https://developers.google.com/identity/openid-connect/openid-connect#obtainuserinfo
+public struct IDToken {
+
+    public let token: String
```

Comment:
My preference is using a strong type if possible, like using `JWToken` instead of `String` here. So that the call site can be sure it's a valid ID token, also makes it impossible to create a `IDToken` instance without a valid JWT.

Reply:
💯 can't believe I missed this. Glad you spotted it.

Addressed in de81593d4407f9cf0bbbac387741dfc981b58b5d